openworld/sdk/generator/client/visitors/operations.py

```diff
@@ -11,21 +11,64 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import collections
+import re
 from pathlib import Path
 
+from datamodel_code_generator.model import DataModel
 from datamodel_code_generator.model.pydantic import CustomRootType
-from fastapi_code_generator.parser import OpenAPIParser, Operation
+from fastapi_code_generator.parser import Argument, OpenAPIParser, Operation
 from fastapi_code_generator.visitor import Visitor
 
 
-def get_operations(parser: OpenAPIParser, model_path: Path) -> dict[str, object]:
-    sorted_operations: list[Operation] = []
+def parse_non_schema_models(parser: OpenAPIParser):
+    models: list[DataModel] = list(
+        {
+            result
+            for result in parser.results
+            if isinstance(result, DataModel)
+            and result.reference.path.startswith("#/paths/")
+            and not result.class_name.endswith("Response")
+            and not result.class_name.endswith("Request")
+        }
+    )
+
+    models_classnames = [model.class_name for model in models]
+    for index in range(len(models) - 1, -1, -1):
+        model = models[index]
+        if model.class_name[-1].isdigit() or f"{model.class_name}1" in models_classnames:
+            continue
+        models.pop(index)
+        index += 1
+
+    return models
+
+
+def clean_unused_models(parser: OpenAPIParser, unused_models_classnames: list[str]):
+    indices_stack = list()
```

Comment:
We could use the `filter` function to simplify the function to one-liner. ([read this](https://www.programiz.com/python-programming/methods/built-in/filter)).

Reply:
Done.